xml/deployment_preparation.xml

```diff
@@ -56,21 +56,32 @@
    </listitem>
   </varlistentry>
   <varlistentry>
-   <term><xref linkend="sec.deploy.prepare.disk_images" /></term>
+   <term><xref linkend="sec.deploy.preparation.disk_images" /></term>
    <listitem>
     <para>
      You can use prepared disk images ready to deploy on virtual
      machines. This increases the deployment speed.
     </para>
    </listitem>
   </varlistentry>
+  <varlistentry>
+   <term><xref linkend="sec.deploy.preparation.public_cloud" /></term>
+   <listitem>
+    <para>
+     This section is about installing &productname; in a public cloud,
+     for example <emphasis>Microsoft Azure</emphasis>*,
+     <emphasis>Amazon AWS</emphasis>* and <emphasis>Google Compute
+     Engine</emphasis>*.
+    </para>
+   </listitem>
+  </varlistentry>
  </variablelist>
  <para>
   To customize the setup of the nodes, use <command>cloud-init</command>.
   For details, see <xref linkend="sec.deploy.cloud-init" />.
  </para>
 
- <sect1 xml:id="sec.deploy.prepare.dvd">
+ <sect1 xml:id="sec.deploy.preparation.dvd">
   <title>Installing from USB, DVD or ISO Images</title>
```

Comment:
Please swap around the order of the sections so it's:

* USB,DVD, ISO
* Virtual Disk
* Network Source
* Openstack Cloud
* Public Cloud 

This groups the "types" better